### DISCUSSION TITLE
How is `SEE` affected by word-lists?

### DISCUSSION THREAD

**Discussion by SirWumpus** [ForthHub/discussion#180] [Q&A] [answered]
The description of `SEE` in the Forth draft section 15.6.1.2194 makes no mention of word-lists, whether it searches all word-lists (assumed to be the case), just the current compilation list, or confined by search order.

What is the expected behaviour WRT word-lists?

**Comment by MitchBradley** [comment]
In my implementations, SEE respects the search order in the same way as the interpreter.

**Comment by ruv** [comment, marked as answer]
A common expectation is that the Forth system resolves word names using **the search order** (if one exists, or just the dictionary otherwise). Thus `see` should resolve names just like «`'`» ([Tick](https://forth-standard.org/standard/core/Tick)). Probably, the glossary entry [15.6.1.2194 `SEE`](https://forth-standard.org/standard/tools/SEE) should be clarified in this regard.

If a Forth system provides the Search-Order word set, it shall respect [16.3.3 Finding definition names](https://forth-standard.org/standard/search#search:find) in addition to [3.4.2 Finding definition names](https://forth-standard.org/standard/usage#usage:find).
